src/experiment/procedures/honeycombProcedure.js

```diff
@@ -39,12 +38,13 @@ export const buildHoneycombProcedure = () => {
     choices: honeycombSettings.timeline_variables.map((variable) => variable.correct_response),
     data: {
       // Record the correct_response passed as a timeline variable
-      code: eventCodes.honeycomb,
+      code: honeycombSettings.code,
```

Comment:
since `honeycombSettings` is set to `settings.honeycomb`, we can directly access the `code` field